main/src/main/scala/sbt/internal/nio/CheckBuildSources.scala

```diff
@@ -26,11 +26,13 @@ private[sbt] object CheckBuildSources {
         (changedInputFiles in checkBuildSources).value match {
           case Some(cf: ChangedFiles) if !firstTime =>
             val rawPrefix = s"Meta build source files have changed:\n" +
-              (if (cf.created.nonEmpty) s"creations: ${cf.created.mkString("\n  ", "  \n", "\n")}"
+              (if (cf.created.nonEmpty) s"New files: ${cf.created.mkString("\n  ", "  \n", "\n")}"
```

Comment:
I worked on a bunch of compiler error messages for Scala 2.13, and I've unified them all to start with lower case. Classic compiler errors like "cannot find symbol" are set in lowercase. I've been thinking we should start doing that for sbt as well for it's logs.

Reply:
Ok. I care more about "new files" vs. "creations". If I drop the upper case here, then I should drop it on the rawPrefix part too I think.